pkgs/applications/virtualization/distrobox/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "distrobox";
-  version = "1.5.0.2";
+  version = "1.6.0.1";
 
   src = fetchFromGitHub {
     owner = "89luca89";
     repo = finalAttrs.pname;
```

```suggestion
    repo = "distrobox";
```
This is wrong. If we append some suffix to distrobox, this would break the FOD.